src/material-experimental/mdc-button/button-base.ts

```diff
@@ -124,6 +111,14 @@ export class MatButtonBase extends _MatButtonBaseMixin implements CanDisable, Ca
   private _hasHostAttributes(...attributes: string[]) {
     return attributes.some(attribute => this._elementRef.nativeElement.hasAttribute(attribute));
   }
+
+  _getHostElement() {
```

Comment:
not needed since `_elementRef` is public, just do `[matRippleTrigger]="_elementRef.nativeElement"`

Reply:
Aye - this is how we did it in MatButton but not sure. Just copied it over to stay the same, but I don't see the benefit to it